### PR TITLE
Moved components to updated locations frontend

### PR DESCRIPTION
**Pull request by KalindaJ-debug**

### CODE REVIEW

File: frontEnd/src/app/core/interfaces/auth/register-request.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface IRegisterRequestInput {
+    name: string;
+    email: string;
+    password: string;
+}
```
Comment (sudarshanadayananda) [inline frontEnd/src/app/core/interfaces/auth/register-request.ts]: @KalindaJ-debug 
You can generalize `ILoginRequestInput ` and `IRegisterRequestInput `.  as follows. 

```
interface IAuthRequestInput {
    email: string;
    password: string;
}

interface IRegisterRequestInput extends IAuthRequestInput {
  name: string;
}
```

But your one is also good. you can keep your preferred one.

File: frontEnd/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
-    component:LoginComponent,
-    path:'login'
+    component:HomepageComponent,
+    path: ''
   },
   {
-    component:RegisterComponent,
-    path: 'register'
+    path: 'login',
+    component: LoginComponent,
```
Comment (sudarshanadayananda) [inline frontEnd/src/app/app-routing.module.ts]: No need to lozy load `AuthModule` twice.

It should be like below

```
 {
    path: 'auth',
    loadChildren: () => import('./modules/auth/auth.module').then(m => m.AuthModule)
  },
```

remove the `component` attribute from route object.

**Comment by KalindaJ-debug** [comment]
Okay will do

**Comment by KalindaJ-debug** [comment]
OKay Changes made ans the code will be pushed

File: frontEnd/src/app/core/interfaces/auth/login-request.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export interface IAuthRequestInput {
```
Comment (sudarshanadayananda) [inline frontEnd/src/app/core/interfaces/auth/login-request.ts]: interfaces should not be in **core** folder.
move **interfaces** folder to a folder like **src/app/shared/interfaces**.

**Comment by KalindaJ-debug** [comment]
Closed for new pull request

**Pull request closed without merging**